scripts/update_deps.py

```diff
@@ -43,6 +43,9 @@
 _JAX_PROJECT_URL = _JAX_INDEX_URL + 'jax/'
 _JAXLIB_PROJECT_URL = _JAX_INDEX_URL + 'jaxlib/'
 
+_TORCH_COMMIT_FORMAT = "# commit %H%n# Author: %an <%ae>%n# Author Date:    %ad%n# Committer Date: %cd%n#%n#    %s%n#%n%H"
```

Comment:
Let's just keep the date in the comment. The other parts of the comment actually distract and have negative value.